src/.pylintrc.jinja

```diff
@@ -27,5 +27,8 @@
     redefined-builtin,
     too-complex,
     unnecessary-utf8-coding-comment
+{%- if odoo_version >= 18 %},
+    prefer-env-translation
+{%- endif %}
```

Comment:
It seems the rest of checks are alphabetically-sorted. Could you do the same here please?